readme.md

```diff
@@ -104,6 +104,7 @@ Click the type names for complete docs.
 - [`Entries`](source/entries.d.ts) - Create a type that represents the type of the entries of a collection.
 - [`SetReturnType`](source/set-return-type.d.ts) - Create a function type with a return type of your choice and the same parameters as the given function type.
 - [`Asyncify`](source/asyncify.d.ts) - Create an async version of the given function type.
+- [`AsExtendable`](source/as-extendable.d.ts) - Wrap a type like `JSX.IntrinsicElements["a"]` so that it could be used to extend an interface.
```

Comment:
> like `JSX.IntrinsicElements["a"]`

This might make the user think only that kind of type works. I would prefer to make the description without an exact example.